packages/destination-actions/src/destinations/drip/identify/index.ts

```diff
@@ -0,0 +1,111 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+const person = (payload: Payload) => {
+  return {
+    custom_fields: payload.customFields,
+    email: payload.email,
+    ip_address: payload.ip,
+    new_email: payload.newEmail,
+    sms_number: payload.sms,
+    status: payload.status,
+    status_updated_at: payload.statusUpdatedAt,
+    tags: payload.tags,
```

Comment:
This needs to format the tags into an array, right? This is something worth talking to Thomas about. Like, ideally we'd format before displaying to the user as the expected payload.

Reply:
Done